ContactUI.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>

```

Comment:
Some naming convention that we need to follow here :slightly_smiling_face: 
1- Instead of ContactUI.html, I suggest to rename it to index.html, since this is the main container page for your single page app.
2- It's a good practice to include any scripts at the end of your html file, like you did with "ContactControles.js".
3- Please rename ContactControles.js to contact-controllers.js :slightly_smiling_face: